<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TarMass</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icon.png</normaloff>icons/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="controlTab">
         <attribute name="title">
          <string>Control</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0">
             <widget class="QComboBox" name="detectionComboBox">
              <item>
               <property name="text">
                <string>YOLOv8m pose</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>keypoint RCNN</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QComboBox" name="scaleComboBox">
              <item>
               <property name="text">
                <string>find-contours + heuristics</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QComboBox" name="interpolationComboBox">
              <item>
               <property name="text">
                <string>quadratic</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cubic</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>slinear</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="interpolationLabel">
              <property name="text">
               <string>Line interpolation algorithm</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="detectionAlgLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Keypoints detecion algorithm</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="inferenceButton">
              <property name="text">
               <string>Inference</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/geolocation.png</normaloff>icons/geolocation.png</iconset>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="scaleAlgLabel">
              <property name="text">
               <string>Scale detection algorithm</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="calculateButton">
              <property name="text">
               <string>Measure</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/calculator.png</normaloff>icons/calculator.png</iconset>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QPushButton" name="clearButton">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="reportButton">
              <property name="text">
               <string>Generate report</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/report.png</normaloff>icons/report.png</iconset>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="stopButton">
              <property name="text">
               <string>Stop process</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/minus-circle.png</normaloff>icons/minus-circle.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QTextEdit" name="textEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="acceptRichText">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="correctionToolTab">
         <attribute name="title">
          <string>Correction tool</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="previousButton">
            <property name="text">
             <string>Previous</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/arrow-180.png</normaloff>icons/arrow-180.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="openImageButton">
            <property name="text">
             <string>Open selcted image</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/document.png</normaloff>icons/document.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Scale value [ Âµm ]</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="nextButton">
            <property name="text">
             <string>Next</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/arrow.png</normaloff>icons/arrow.png</iconset>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>File list</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QListWidget" name="imagesListWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="scaleSpinBox">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="instanceButton">
            <property name="text">
             <string>Create instance</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Dir"/>
    <addaction name="actionChange_Save_Dir"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/question.png</normaloff>icons/question.png</iconset>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Dir">
   <property name="icon">
    <iconset>
     <normaloff>icons/folder.png</normaloff>icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Dir</string>
   </property>
  </action>
  <action name="actionChange_Save_Dir">
   <property name="icon">
    <iconset>
     <normaloff>icons/folder.png</normaloff>icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Change Save Dir</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>icons/disk.png</normaloff>icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>icons/disk.png</normaloff>icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAppIcon">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../../../.designer/backup/icon.jpg</normaloff>../../../../../../../../.designer/backup/icon.jpg</iconset>
   </property>
   <property name="text">
    <string>AppIcon</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>User Manual</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="fugue-icons-3.5.6/icons/arrow-180.png"/>
 </resources>
 <connections/>
</ui>
